datacenter/ucp/2.2/guides/admin/install/system-requirements.md

```diff
@@ -9,23 +9,34 @@ Before installing, be sure your infrastructure has these requirements.
 
 ## Hardware and software requirements
 
-You can install UCP on-premises or on a cloud provider. To install UCP,
-all nodes must have:
+You can install UCP on-premises or on a cloud provider. Common requirements:
 
 * [Docker Enterprise Edition](/engine/installation/index.md) version 17.06 or higher
 * Linux kernel version 3.10 or higher
+* A static IP address
+
+### Minimum requirements
+
 * 8GB of RAM for manager nodes or nodes running DTR
 * 4GB of RAM for worker nodes
 * 3GB of free disk space
-* A static IP address
+
+### Recommended production requirements
+
+ * 16GB of RAM for manager nodes or nodes running DTR
```

Comment:
16 GB should work with any large instance.  For POCs, we can recommend 2 GB of RAM

Reply:
@pdevine does it make sense to mention DTRs running the security scanner separately?